packages/test-runner-visual-regression/src/visualRegressionPlugin.ts

```diff
@@ -67,7 +67,13 @@ export function visualRegressionPlugin(
             }
 
             const screenshot = (await element.screenshot({ encoding: 'binary' })) as Buffer;
-            return visualDiffCommand(mergedOptions, screenshot, session.browser.name, payload.name);
+            return visualDiffCommand(
+              mergedOptions,
+              screenshot,
+              session.browser.name,
+              payload.name,
+              session.testFile,
```

Comment:
Feels like the argument list is getting a little unwieldy here, but I just made the smallest addition I could. Didn't want to start restructuring things!

Reply:
Thanks for the reasonable approach. I think you have a valid point here about the argument list.

What if we change `visualDiffCommand` to accept `context` object as a third argument?
It could contain `browser` and `testFile` fields for now and maybe something else later. 

In that case we migth want to also export `VisuallDiffCommandContext` interface.

Reply:
Sorry, I have minimized the above comment because it appears to be shown two times for me 😅

Reply:
haha no worries. I am thinking of also adding the `payload.name` into the new Context type, since it is used everywhere browser and testFile are used? e.g. as it is now:

```js

options.getBaselineName({ browser, name, testFile })
```

could become:

```js
options.getBaselineName(context)
```

what do you think?

Reply:
Though by then the type is identical to `GetNameArgs` haha. Seems like there's some overlap in intent

Reply:
Maybe it's best to stick with your initial suggestion

Reply:
I think that would be a breaking change, while my suggestion isn't.